azurerm/internal/services/cosmos/cosmosdb_sql_container_resource.go

```diff
@@ -78,7 +78,6 @@ func resourceArmCosmosDbSQLContainer() *schema.Resource {
 			"default_ttl": {
 				Type:         schema.TypeInt,
 				Optional:     true,
-				Computed:     true,
 				ValidateFunc: validation.IntAtLeast(-1),
```

Comment:
Could we also add a `Default: -1` here?
Because I suppose if the user does not specify this attribute, he/she would mean absence of this attribute which should be equivalent to `disable`.
In the current implementation, if you leave this empty, terraform would assign `0` for you.

Reply:
We cannot do this as it will break existing users who depend on the computed value in there state.

Reply:
@ArcturusZhang Agree. @katbyte But the field is not `computed`. The api will not return a default value if this field is absent in request. The current computed value should be `0` because this field is absent in request. In this case, shall we remove the `computed`?